src/NServiceBus.AcceptanceTests/Recoverability/When_custom_policy_does_single_delayed_retry_before_move_to_error.cs

```diff
@@ -11,21 +12,28 @@ namespace NServiceBus.AcceptanceTests.Recoverability.Retries
     public class When_custom_policy_does_single_delayed_retry_before_move_to_error : NServiceBusAcceptanceTest
     {
         [Test]
-        public async Task Should_execute_twice()
+        public async Task Should_execute_twice_and_send_to_error_queue()
         {
+            var messageId = Guid.NewGuid().ToString();
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<RetryEndpoint>(b => b
-                    .When(bus => bus.SendLocal(new MessageToBeRetried()))
+                    .When(bus =>
+                    {
+                        var sendOptions = new SendOptions();
+                        sendOptions.RouteToThisEndpoint();
+                        sendOptions.SetMessageId(messageId);
+                        return bus.Send(new MessageToBeRetried(), sendOptions);
+                    })
                     .DoNotFailOnErrorMessages())
-                .Done(c => c.MessageSentToErrorQueue)
+                .Done(c => c.FailedMessages.Any())
                 .Run();
 
             Assert.AreEqual(context.Count, 2);
```

Comment:
As part of this PR, do we want to ensure all Assert parameters are in the correct order, or leave that for another PR, because this one is incorrect.